src/swift/basetypes/MCOIndexSet.swift

```diff
@@ -20,12 +20,13 @@ public class MCOIndexSet: NSObjectCompat, NSCopying, NSCoding {
         self.nativeInstance = CIndexSet_init();
     }
     
-    init?(_ indexSet: CIndexSet) {
+    init?(_ indexSet: CIndexSet, autoreleasePool: CAutoreleasePool? = nil) {
         guard  indexSet.instance != nil else {
             return nil
         }
         self.nativeInstance = indexSet;
         self.nativeInstance.retain()
+        autoreleasePool?.release()
```

Comment:
and why do we release it in init just after it was passed in?

Reply:
`indexSet` was added to this autorelease pool, we should release this pool after retain
I guess we can rewrite in a more elegant way